SQL/Archive/19.2/2018-06-05_Add_UUID_column_to_document_repository.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE document_repository ADD `UUID` varchar(36) DEFAULT '';
```

Comment:
maybe this patch should go into just the Archive directory? we're now on the 20 release. also, patches usually get moved to the version folders after a release if on minor

Reply:
While we are at this, when would you submit the patches into the `New_patches` dir?

Reply:
@um4r12 when you are on a branch that has these new directories. i'm going to bring this up in tomorrow's meeting :)